src/config/configureMocks.js

```diff
@@ -1,11 +1,14 @@
 import MockAdapter from 'axios-mock-adapter';
 import authAdapter from './mocks/adapters/auth';
 import exampleAdapter from './mocks/adapters/example';
+import dinersAdapter from './mocks/adapters/diners';
+
 export const isMockEnabled = () => {
   return process.env.REACT_APP_MOCK_ENABLED === 'true';
 };
 export const initializeAxiosMockAdapter = (instance) => {
   const mock = new MockAdapter(instance, { delayResponse: 550 });
   authAdapter.init(mock);
```

Comment:
Can explain what the mock does?

Reply:
Sure, the axios-mock-adapter what it do is when there is a request using the axios client, it will review the PATH to the request and look for an adapter. 

If the adapter if present on the configuration, it will execute the function on the adapter instead of making the HTTP request to the server.

This is helpful when you don't have a backend ready to mock the request/response of the Backend and build it later so you don't have to put fake data inside the components of the React App.